react-native-picker-module.d.ts

```diff
@@ -20,6 +20,8 @@ declare module "react-native-picker-module" {
     hide: () => void,
   }
 
+  export type Item = number | string | { label: string; value: number | string }
```

Comment:
Looking at the android code, I think `items` could have a more specific type than `any`

```
        if (mDataset.getType(position) == ReadableType.Map) {
            if (mDataset.getMap(position).getType("value") != ReadableType.Null) {
                if (mDataset.getMap(position).getType("value") == ReadableType.String) {
                    value = mDataset.getMap(position).getString("value");
                } else {
                    double number = mDataset.getMap(position).getDouble("value");
                    if (number == Math.rint(number)) {
                        value = String.valueOf((int) number);
                    } else {
                        value = String.valueOf(number);
                    }
                }
                if (mDataset.getMap(position).getType("label") == ReadableType.String) {
                    text = mDataset.getMap(position).getString("label");
                } else {
                    double number = mDataset.getMap(position).getDouble("label");
                    if (number == Math.rint(number)) {
                        text = String.valueOf((int) number);
                    } else {
                        text = String.valueOf(number);
                    }
                }
            }
        } else if (mDataset.getType(position) == ReadableType.String) {
            text = mDataset.getString(position);
            value = mDataset.getString(position);
        } else if (mDataset.getType(position) == ReadableType.Number) {
            double number = mDataset.getDouble(position);
            if (number == Math.rint(number)) {
                text = String.valueOf((int) number);
                value = String.valueOf((int) number);
            } else {
                text = String.valueOf(number);
                value = String.valueOf(number);
            }
        }